<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>205</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>594</width>
    <height>205</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>594</width>
    <height>205</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>登录火车订票系统</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>12308.png</normaloff>12308.png</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>用户名</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QPushButton" name="exitBtn">
     <property name="minimumSize">
      <size>
       <width>576</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>576</width>
       <height>31</height>
      </size>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="auloginBtn">
     <property name="minimumSize">
      <size>
       <width>83</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>83</width>
       <height>31</height>
      </size>
     </property>
     <property name="text">
      <string>管理员登录</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="loginBtn">
     <property name="minimumSize">
      <size>
       <width>487</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>487</width>
       <height>31</height>
      </size>
     </property>
     <property name="text">
      <string>用户登录</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="pwdLineEdit">
     <property name="minimumSize">
      <size>
       <width>487</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>487</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="usrLineEdit">
     <property name="minimumSize">
      <size>
       <width>487</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>487</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="registBtn">
     <property name="minimumSize">
      <size>
       <width>576</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>576</width>
       <height>31</height>
      </size>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitBtn</sender>
   <signal>clicked()</signal>
   <receiver>Login</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>usrLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>loginBtn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>482</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pwdLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>loginBtn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>482</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
